src/solvers/dg/2d/dg.jl

```diff
@@ -1216,6 +1216,10 @@ function rhs!(dg::Dg2D, t_stage)
   @timeit timer() "source terms" calc_sources!(dg, dg.source_terms, t_stage)
 end
 
+# TODO: implement 2D!!!
+# Apply positivity limiter of Zhang and Shu to nodal values elements.u
+function apply_positivity_preserving_limiter!(dg::Dg2D)
+end
```

Comment:
I think we should not have a method like this that is not implemented. Instead, it would be better to open it as an issue (good first issue!) and just delete it IMHO.
```suggestion
```

Reply:
we need this, such that the 2D runs at all...
otherwise, we need to change the structure back, as I had it...